MissionBit/BAMPWebsite
# Add garden video clip to homepage carousel

Adds a garden video to the homepage.

![Jan-24-2021 18-35-51](https://user-images.githubusercontent.com/733916/105654567-20d4d080-5e73-11eb-9e15-bf520b0df210.gif)

~~note: this causes problems on my phone and will probably need to be hidden on mobile.~~

update: changed the video to a webp "image." works fine on my iphone now


### to test on mobile:

- in vscode settings, search for "live server host" and change the host to `0.0.0.0`
- make sure you don't have a firewall on, or that you are allowing incoming traffic to port 5500
- run `ipconfig` (windows) or `ifconfig` (mac / linux) and look for your local IP address. should start with 192
- hit "go live" in vscode
- open `http://<your-desktop-ip-address>:5500` on your phone
